archeo/inc/patterns/simple-list-of-posts-with-background.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Simple list of posts with title and background
+ */
+return array(
+	'title'      => __( 'Simple list of posts with title and background', 'archeo' ),
+	'categories' => array( 'pages' ),
+	'content'    => '<!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"var(--wp--custom--spacing--medium)","bottom":"var(--wp--custom--spacing--medium)"},"margin":{"top":"0px"}}},"backgroundColor":"foreground","textColor":"background"} -->
+	<div class="wp-block-group alignwide has-background-color has-foreground-background-color has-text-color has-background" style="padding-top:var(--wp--custom--spacing--medium);padding-bottom:var(--wp--custom--spacing--medium);margin-top:0px">
+		<!-- wp:heading {"level":3,"style":{"spacing":{"margin":{"bottom":"60px"}}},"fontSize":"medium"} -->
+		<h3 class="has-medium-font-size" style="margin-bottom:60px">' . __( 'Selected Writings', 'archeo' ) . '</h3>
+		<!-- /wp:heading -->
+		
+		<!-- wp:query {"queryId":3,"query":{"pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":false,"perPage":3}} -->
+		<div class="wp-block-query"><!-- wp:post-template -->
+		<!-- wp:group {"className":"simple-list-of-posts","layout":{"type":"flex","allowOrientation":false}} -->
+		<div class="wp-block-group simple-list-of-posts"><!-- wp:post-title {"style":{"typography":{"lineHeight":"1.2","fontStyle":"normal","fontWeight":"100"}},"fontSize":"huge"} /-->
+		
+		<!-- wp:post-author-name {"style":{"typography":{"fontStyle":"normal","fontWeight":"100","lineHeight":"1.2"}},"fontSize":"medium"} /--></div>
+		<!-- /wp:group -->
+		<!-- /wp:post-template --></div>
+		<!-- /wp:query --></div>
```

Comment:
I think this needs pagination. For now, let's just add prev/next links, with no page numbers. The text size can match our nav's text size.

Reply:
Great ok, I've added pagination now too:

<img width="1424" alt="image" src="https://user-images.githubusercontent.com/1645628/154998577-873dab63-e479-48e5-ba71-2d9de60b3692.png">